compiler/pkg/generator/template/graphql/graphqlResolver.go.tmpl

```diff
@@ -65,6 +65,8 @@ func getRootResolver() (*model.{{$node.PkgName}}{{$node.NodeName}}, error) {
             return nil, fmt.Errorf("failed to get k8s client config: %s", err)
 	    }
 	nc = nexusClient
+	nc.SubscribeAll()
+	log.Debugf("Subscribe api is called for all the nodes.")
 }
```

Comment:
This brace is not indented right. Can we indent it.

Reply:
yes, I have updated the code PTAL